03-transactions.md

```diff
@@ -395,25 +469,32 @@ committed HTLCs:
    in [Commitment Transaction](#commitment-transaction).
 1. Calculate which committed HTLCs need to be trimmed (see [Trimmed Outputs](#trimmed-outputs)).
 2. Calculate the base [commitment transaction fee](#fee-calculation).
-3. Subtract this base fee from the funder (either `to_local` or `to_remote`),
-   with a floor of 0 (see [Fee Payment](#fee-payment)).
-3. For every offered HTLC, if it is not trimmed, add an
+3. Subtract this base fee from the funder (either `to_local` or `to_remote`).
+If `option_anchor_outputs` applies to the commitment transaction,
+also subtract two times the fixed anchor size of 330 sats from the funder
+(either `to_local` or `to_remote`).
+4. For every offered HTLC, if it is not trimmed, add an
    [offered HTLC output](#offered-htlc-outputs).
-4. For every received HTLC, if it is not trimmed, add an
+5. For every received HTLC, if it is not trimmed, add an
    [received HTLC output](#received-htlc-outputs).
-5. If the `to_local` amount is greater or equal to `dust_limit_satoshis`,
+6. If the `to_local` amount is greater or equal to `dust_limit_satoshis`,
    add a [`to_local` output](#to_local-output).
-6. If the `to_remote` amount is greater or equal to `dust_limit_satoshis`,
+7. If the `to_remote` amount is greater or equal to `dust_limit_satoshis`,
    add a [`to_remote` output](#to_remote-output).
-7. Sort the outputs into [BIP 69+CLTV order](#transaction-input-and-output-ordering).
+8. If `option_anchor_outputs` applies to the commitment transaction:
+   * if `lo_local` exists and/or there are HTLCs, add a `to_local_anchor` output
```

Comment:
```suggestion
   * if `to_local` exists and/or there are HTLCs, add a `to_local_anchor` output
```

Reply:
Github shows two identical sentences. Marked resolved for now.

Reply:
you have `lo_local`, it should be `to_local` ??